benchmarks/source-strapi/package.json

```diff
@@ -14,7 +14,7 @@
     "prop-types": "^15.7.2",
     "react": "^16.12.0",
     "react-dom": "^16.12.0",
-    "sharp": "^0.25.4"
+    "sharp": "^0.26.1"
```

Comment:
This is breaking our tests due to node requirements. I believe there was some discussion about this bump?

Reply:
I'm going to take a look at fixing Sharp's versioning https://github.com/lovell/sharp/issues/2410

Reply:
OK, I have a PR: https://github.com/lovell/sharp/pull/2412